### PR TITLE
Added message to component interaction

### PR DESCRIPTION
**Pull request by HarryET**
Added the message cacheable to the interaction for component.

### CODE REVIEW

**Review by l7ssha — CHANGES_REQUESTED** [review]
Since interactons are available in DMs implementation needs to take that into consideration

**Review by l7ssha — APPROVED** [review]

File: nyxx/lib/src/utils/EntityUtility.dart [hunk 1]
```diff
@@ -102,4 +102,28 @@ class EntityUtility {
...
+  /// ```dart
+  /// void main() {
+  ///     var bot = Nyxx("TOKEN");
+  ///     Map<String, dynamic> rawJson = /* rawJson from the API */
+  ///     DMChannel dmChannel = EntityUtility.createGuildMessage(bot, rawJson);
+  /// }
+  /// ```
+  static GuildMessage createGuildMessage(
+      INyxx client, Map<String, dynamic> rawJson) =>
+      GuildMessage._new(client, rawJson);
```
Comment (l7ssha) [inline nyxx/lib/src/utils/EntityUtility.dart:116]: its pretty much useless if there is createMessage

File: nyxx_interactions/pubspec.yaml [hunk 2]
```diff
@@ -12,3 +12,7 @@ environment:
 dependencies:
   logging: "^1.0.1"
   nyxx: "2.0.0-rc.5"
+
+dependency_overrides:
+  nyxx:
+    path: "../nyxx"
```
Comment (l7ssha) [inline nyxx_interactions/pubspec.yaml:18]: we dont need that

**Review by l7ssha — CHANGES_REQUESTED** [review]

File: nyxx_interactions/pubspec.yaml (hunk 2 above)
Comment (HarryET) [reply nyxx_interactions/pubspec.yaml]: done

File: nyxx/lib/src/utils/EntityUtility.dart (hunk 1 above)
Comment (HarryET) [reply nyxx/lib/src/utils/EntityUtility.dart]: done

**Review by l7ssha — APPROVED** [review]

**Pull request merged**